src/mots/utils.py

```diff
@@ -37,7 +37,8 @@ def parse_real_name(real_name):
     pattern = re.compile(r"^(?P<name>[\w\ ]+?)?\ ?(?P<info>[\(\[\|\:].*)?$")
     match = pattern.match(real_name)
     if match:
-        return match.groupdict()
+        data = match.groupdict()
+        return {k: v.strip() if v else "" for k, v in data.items()}
```

Comment:
The below `None`s should be empty strings, will update. The rationale behind switching from `None` to `""` is readability in the yaml file (since otherwise there will be a bunch of `null`s in there.

Reply:
Moreover, the code path below is never executed, since `bool(match)` is always `True`, and `match.groupdict` will return null values with the right keys regardless. Will fix.